### PR TITLE
storaged: add Arch Linux nfs-utils/stratisd packages

### PR DESCRIPTION
**Pull request by jelly**
Make it possible to install optional NFS/Stratis support using
PackageKit.

### CODE REVIEW

**Comment by mvollmer** [comment]
The TestStoragePackagesNFS.testNfsMissingPackages is skipped in arch: "TODO: bug in refreshing removed/installed state on Arch Linux".  So does this work at all?  Would be great to get the test working as well.

**Comment by jelly** [comment]
> The TestStoragePackagesNFS.testNfsMissingPackages is skipped in arch: "TODO: bug in refreshing removed/installed state on Arch Linux". So does this work at all? Would be great to get the test working as well.

Excellent idea, enabled the test :eye:

**Comment by jelly** [comment]
This worked fine when running just the test, so it seems that something is getting in the way

**Comment by jelly** [comment]
So pkcon can install the package fine, so it seems to be something with the dbus api which is broked?

File: test/verify/packagelib.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def setUp(self):
             self.restore_file("/etc/pacman.d/mirrorlist")
             self.restore_file("/usr/share/libalpm/hooks/90-packagekit-refresh.hook")
             self.machine.execute("rm /etc/pacman.conf /etc/pacman.d/mirrorlist /var/lib/pacman/sync/* /usr/share/libalpm/hooks/90-packagekit-refresh.hook")
+            self.machine.execute("test -d /var/lib/PackageKit/alpm && rm -r /var/lib/PackageKit/alpm || true")  # Drop alpm state directory as it interferes with running offline
```
Comment (marusak) [inline test/verify/packagelib.py:74]: Do we need to test for it when you do `|| true` anyway?

Comment (jelly) [reply test/verify/packagelib.py:74]: As when it does not exists test returns "false" which makes execute fail.

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**